directives/questionnaire.py

```diff
@@ -35,6 +35,7 @@ class Questionnaire(AbstractExercise):
         'title': directives.unchanged,
         'category': directives.unchanged,
         'status': directives.unchanged,
+        'show-correct-once': choice_truefalse,
```

Comment:
Would `directives.flag` work here if the default is False and the True value is only written to the config YAML if the directive has the option flag set?

Reply:
But if the teacher wants to set default to the course that model answers are shown but wants to prevent that for some specific exercises, there should be an easy way to do that, too.

Reply:
OK. Let's move the default setting from conf.py to the field list of the index.rst, though.